test/funcfile-docker-rt-tests/testfiles/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM golang:latest
```

Comment:
I'd recommend to use something smaller like `golang alpine`.

Reply:
I tried with 'FROM golang:alpine', it appears took looong time pull it on my system when running the test, so didn't change.